lib/game.rb

```diff
@@ -0,0 +1,82 @@
+require 'pry'
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    @board.turn_count.even? ? player_1 : player_2
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |combination|
+      if @board.cells[combination[0]] != " "
+        if @board.cells[combination[0]] == @board.cells[combination[1]] && @board.cells[combination[1]]== @board.cells[combination[2]]
+           return combination
+        end
+      end
+    end
+    false
+  end
+
+  def over?
+    won? || draw?
+  end
+
+  def draw?
+    @board.full? && !won?
+  end
+
+  def winner
+    win_combo = won?
+     if win_combo
+       if @board.cells[win_combo[0]] == " "
+           nil
```

Comment:
You do not need to explicitly return `nil` from an `if` statement; `if` will return `nil` if the condition fails.
```
if @board.cells[win_combo[0]] != " "
  @board.cells[win_combo[0]]
end
```